src/tkpysdk/__init__.py

```diff
@@ -1 +1 @@
-from .signature import *
\ No newline at end of file
+from tkpysdk.utils.signature import *
```

Comment:
I believe you have to use relative path here.
export other new packages besides signature